django_cloud_deploy/cloudlib/static_content_serve.py

```diff
@@ -44,6 +44,31 @@ def __init__(self, storage_service: discovery.Resource):
     def from_credentials(cls, credentials: credentials.Credentials):
         return cls(discovery.build('storage', 'v1', credentials=credentials))
 
+    def _bucket_exist(self, project_id: str, bucket_name: str) -> bool:
+        """Returns whether the given bucket exists under the given project.
+
+        Args:
+            project_id: Id of the GCP project.
+            bucket_name: Name of the bucket.
+
+        Returns:
+           Whether the given bucket exists under the given project.
+
+        Raises:
+            StaticContentServeError: When it fails to list buckets under the
+                given project.
+        """
+        request = self._storage_service.buckets().list(project=project_id)
+        response = request.execute()
+        if 'items' not in response:
+            raise StaticContentServeError(
+                'Unexpected response listing buckets in project "{}"'
+                ': {}'.format(project_id, response))
+        for item in response['items']:
```

Comment:
Just an FYI, you can write this as:

return any(item['name'] == bucketname for item in response['items'])